linkedin_agent.py

```
@@ -0,0 +1,444 @@
+        # Cargar datos existentes
+        self.scheduled_posts = self._load_data(self.posts_file, [])
+        self.connections = self._load_data(self.connections_file, [])
+        self.analytics = self._load_data(self.analytics_file, {})
+
+    def _load_data(self, filename: str, default):
+        """Carga datos desde archivo JSON"""
+        if os.path.exists(filename):
+            try:
+                with open(filename, 'r', encoding='utf-8') as f:
+                    return json.load(f)
+            except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)